src/models/UrlShorten.js

```diff
@@ -6,7 +6,7 @@ const UrlShortenSchema = new Schema({
   clipped_url: { type: String, required: true },
   urlCode: { type: String, required: true },
   createdAt: { type: Date, default: Date.now },
-  created_by: { type: String, required: true },
+  createdBy: { type: String, required: true },
```

Comment:
this should be `created_by` snake_case not  `createdBy` camelCase. It was changed to snake_case in one of our commits last week.